tools/macos-installer/pkgbuild/npm/scripts/preinstall

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+[[ -d /usr/local/lib/node_modules/npm ]] \
+  && rm -rf /usr/local/lib/node_modules/npm
```

Comment:
This script needs to return an exit code 0 to work ([see docosx](https://matthew-brett.github.io/docosx/flat_packages.html#component-installer)). Otherwise it will abort (showing "The installation failed" if there isn't a `/usr/local/lib/node_modules/npm` ). Can you add

```
exit 0
```

to the end of this file?

Reply:
Sure can

Reply:
Doesn't `rm -f` return 0 even when there was no file to remove? And when it failed to remove npm then we have an unexpected failure anyway.